webtau-feature-testing/src/main/groovy/com/twosigma/webtau/featuretesting/WebTauEndToEndTestRunner.groovy

```diff
@@ -62,15 +56,15 @@ class WebTauEndToEndTestRunner implements StepReporter, StandaloneTestListener {
         StepReporters.add(this)
         StandaloneTestListeners.add(this)
 
-        def reportPath = 'examples/'
-        if (testFileName.endsWith('.groovy')) {
-            reportPath += testFileName.replace('.groovy', '-webtau-report.html')
-        } else {
-            reportPath += testFileName + '/webtau-report.html'
-        }
+        def targetClasses = DocumentationArtifactsLocation.classBasedLocation(WebTauEndToEndTestRunner)
+        def reportPath = targetClasses
+                .resolve(testFileName.endsWith('.groovy') ?
+                        testFileName.replace('.groovy', '-webtau-report.html'):
+                        testFileName + '/webtau-report.html')
 
         try {
             def args = ['--workingDir=examples', '--config=' + configFileName,
+                        '--docPath=' + targetClasses.resolve('doc-artifacts'),
```

Comment:
would this honor a docPath override?

Reply:
No, it is forcing a docPath. Command line override takes precedence.

Reply:
so you're saying it would honor a docPath override.